src/validation/models/routePathValidationModel.ts

```diff
@@ -1,7 +1,8 @@
 const routePathValidationModel = {
-    nameFi: 'required|min:1|max:60|string',
-    // TODO: These length limits are only made up
-    length: 'required|numeric|min:1|max:100000',
+    source: 'required|min:1|max:20|string',
```

Comment:
should this be renamed `origin` to match form names (or other way around)

Reply:
Fixed